### PR TITLE
Introducing the Keystone Blog 🎉 

### PR DESCRIPTION
**Pull request by flexdinesh**
Changes:
- Menu reorg - introduce a dropdown menu to group important links and make space for Blog and Docs on the header.
- `/blog` landing page
- `/blog/[post]` page
- `og:image` auto generated using `@vercel/og`
-  Previous featured news in `/updates` converted to blog posts
- _Latest news_ in `/updates` still exists but all links to the page have been removed from navigation (once we figure out the value for this page, we'll decide where to add it or remove it altogether)
- All mdx removed. All the release mdx in the docs have been moved to relevant release page on GitHub
- Redirects added for all moved pages

Unrelated fixes:
- FIX: Breadcrumbs showing page anchor

Follow up later with another PR:
- mdx is not needed anymore. Should clean it up later.

### CODE REVIEW

File: docs/markdoc/index.ts [hunk 1]
```diff
@@ -33,16 +33,26 @@ class MarkdocValidationFailure extends Error {
...
 
-export async function readDocContent(filepath: string): Promise<DocContent> {
+export type BlogContent = BlogFrontmatter & {
+  content: Tag;
+};
+
+export async function readBlogContent(filepath: string): Promise<BlogContent> {
+  let content = await fs.readFile(filepath, 'utf8');
+  const frontmatter = extractBlogFrontmatter(content);
+  return { content: transformContent(`blog/${filepath}`, content), ...frontmatter };
```
Comment (mitchellhamilton) [inline docs/markdoc/index.ts:49]:
```suggestion
  return { content: transformContent(`docs/${filepath}`, content), ...frontmatter };
```
(docs refers to the top-level directory)

File: docs/components/Breadcrumbs.tsx [hunk 2]
```diff
@@ -11,7 +11,8 @@ type Path = { title: string; href: string };
 export function Breadcrumbs() {
   const router = useRouter();
 
-  const linkPath = router.asPath.split('/');
+  // remove anchor and split path
+  const linkPath = router.asPath.replace(/#(.*)/g, '').split('/');
```
Comment (mitchellhamilton) [inline docs/components/Breadcrumbs.tsx:15]:
```suggestion
  const linkPath = new URL(router.asPath, 'https://keystonejs.com').pathname.split('/');
```

Comment (flexdinesh) [reply docs/components/Breadcrumbs.tsx]: pushed in a new commit with this change

**Review by emmatown — APPROVED** [review]

File: docs/pages/blog/singleton.md [hunk 3]
```diff
@@ -0,0 +1,79 @@
...
+authorName: "Dinesh Pandiyan"
+authorHandle: "https://twitter.com/flexdinesh"
+metaImageUrl: ""
+---
+
+Singleton is one of the most popular community requested features and it is an easy way to work with lists that will only ever have one item like a _website configuration_ or _user settings_.
+
+You describe the list (Eg. _WebsiteConfiguration_) like you describe any other list and add the property `isSingleton: true` to it.
+
+```
```
Comment (mitchellhamilton) [inline docs/pages/blog/singleton.md:14]:
````suggestion
```ts
````

File: docs/pages/blog/singleton.md [hunk 4]
```diff
@@ -0,0 +1,79 @@
...
+        copyrightText: text(),
+      }
+    }),
+  }),
+});
+```
+
+In GraphQL, you can read and write from this list without needing an identifier.
+
+```
```
Comment (mitchellhamilton) [inline docs/pages/blog/singleton.md:32]:
````suggestion
```graphql
````

File: docs/pages/blog/singleton.md [hunk 5]
```diff
@@ -0,0 +1,79 @@
...
+  ) {
+    websiteName
+    copyrightText
+  }
+}
+```
+
+Within the server environment, you can read and write from this list using the default identifier `1`.
+
+```
```
Comment (mitchellhamilton) [inline docs/pages/blog/singleton.md:61]:
````suggestion
```ts
````

**Pull request merged**